lib/Classifiers/Classifier.php

```diff
@@ -125,6 +125,8 @@ public function classifyFiles(string $model, array $queueFiles, int $timeout): \
 		$this->logger->debug('Classifying '.var_export($paths, true));
 
 		$command = [
+			$this->config->getAppValue('recognize', 'nice_binary'),
+			"-" . $this->config->getAppValue('recognize', 'nice_value'),
```

Comment:
Also, if you use IConfig directly instead of the SettingsService you won't get the default values from settingsService, so if nice_value is not set, it will be `''` instead of `0`

Reply:
I had that in mind, it slipped right off :sweat_smile: